src/components/RoomView/gameLog.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+const GameLog = () => {
+  // eslint-disable-next-line no-unused-vars
+  let gameHistory = useSelector((state) => state.game.gameHistory);
+  let gameStatus = useSelector((state) => state.game.status);
+console.log(gameHistory)
+  return (
+    <div>
+      <h3>Game Status</h3>
+      <div>{gameStatus}</div>
+      <h3>Game History</h3>
+      {gameHistory.map((singleHistory, index) => {
+        if (typeof singleHistory === 'object')
+          return (
+            <p key={index}>
+              The clue is {singleHistory.clueString} and you have {singleHistory.clueNumber} guess(es)
```

Comment:
2 minor syntax suggestion -- > 

1. what are your thoughts on making this wording like:
'team X clue is CLUE  for NUMBER guesses'
 instead of 
'you have Y guesses' (because it'll show that on everyones screen, not just the person guessing)

2. is there a way to tie the team number to the clue so that it can be blue/red depending on who submitted it?